tests/lib/rules/no-unpublished-import.js

```diff
@@ -127,12 +127,6 @@ ruleTester.run("no-unpublished-import", rule, {
             options: [{ allowModules: ["electron"] }],
         },
 
-        // Should not fill in the extension
-        {
-            filename: fixture("2/test.js"),
-            code: "import ignore1 from './ignore1';",
-        },
-
```

Comment:
Not sure why this was passing before :grimacing:

`./ignore1` was not published because it was in the `.npmignore` file. So this test should not have passed.